src/README.md

```diff
@@ -0,0 +1,34 @@
+# Python library for ZFM fingerprint sensors
```

Comment:
In my opionion, the base files like readme, copyright+license should always be in the root directory of the repository. Create a sym link instead

Reply:
Sure, but this is a special README for PyPi which I added to keep the original... The problem is that the current README is for Debian especially. PyPi needs a generalized README which is not related to Debian. How should we proceed? Manage two READMEs?

Reply:
Hmm okay i got it. I think it's a bad idea to maintain two files. What do you think about a minimal README for PyPi that basically refers (with link?) to the main repository README. So we do not copy-paste the introduction. But because the "How to use the library" of the main repository README is specific to Debian package paths, the change you made here is good. To sum up, i would put a minimal introduction with link to main README, the "How to use the library" section is perfect, and maybe remove the "Further information" section because it is copy-pasted from main README. What do you think?

Reply:
Yes, that's the way we do it!